Makefile

```diff
@@ -301,8 +301,9 @@ endif
 
 grf: $(GFX_FILES) $(GRF_FILES)
 
+# custom_tags.txt is used by NML
 custom_tags.txt: Makefile.vcs
-	$(_E) "[LNG] $@"
+	$(_E) "[NML] $@"
```

Comment:
Why? NML is not invoked here, see line 317

Reply:
LNG made no sense what-so-ever to me. It might be that NML needs it for that, but in the context of this repo that is just confusing. NML was the closest I could think of, as it prepares a file for NML to use. Other suggestions?

Reply:
I think "LNG" is not that bad. custom_tags.txt is an "include" for the .lng files.